src/reaction.cpp

```diff
@@ -174,32 +223,98 @@ const std::unordered_map<int, std::string> REACTION_NAME_MAP {
   {HEATING_LOCAL, "heating-local"},
 };
 
+std::unordered_map<std::string, int> REACTION_TYPE_MAP;
+
+void initialize_maps()
+{
+  // Add level reactions to name map
+  for (int level = 0; level <= 48; ++level) {
+    if (level >= 1 && level <= 40) {
+      REACTION_NAME_MAP[50 + level] = fmt::format("(n,n{})", level);
+    }
+    REACTION_NAME_MAP[600 + level] = fmt::format("(n,p{})", level);
+    REACTION_NAME_MAP[650 + level] = fmt::format("(n,d{})", level);
+    REACTION_NAME_MAP[700 + level] = fmt::format("(n,t{})", level);
+    REACTION_NAME_MAP[750 + level] = fmt::format("(n,3He{})", level);
+    REACTION_NAME_MAP[800 + level] = fmt::format("(n,a{})", level);
+    if (level <= 15) {
+      REACTION_NAME_MAP[875 + level] = fmt::format("(n,a{})", level);
```

Comment:
the serpent wiki page has these as n,2n reactions. I cant find them on the lanl page I use usually. Can you please send me a link to what you used?

Reply:
Yes, these should be n,2n reactions. Thanks for catching this! The most up-to-date source is the [ENDF-6 formats manual](https://www.osti.gov/biblio/1425114-endf-formats-manual-data-formats-procedures-evaluated-nuclear-data-files).